htdocs/accountancy/supplier/index.php

```diff
@@ -299,10 +299,14 @@
 $sql .= "  AND ffd.product_type <= 2";
 $sql .= " AND ff.entity IN (".getEntity('facture_fourn', 0).")"; // We don't share object for accountancy
 $sql .= " AND aa.account_number IS NULL";
+if (!empty($conf->global->FACTURE_DEPOSITS_ARE_JUST_PAYMENTS)) {
```

Comment:
Currently FACTURE_DEPOSITS_ARE_JUST_PAYMENTS is a deprecated and abandonned option for sales invoice only.
Because it has been deprecated, it was not implemented for supplier invoice. But if we introduce it, we should name the option differently, for exemple FACTURE_SUPPLIER_DEPOSITS_ARE_JUST_PAYMENTS